plugins/UsersManager/templates/index.twig

```diff
@@ -191,7 +209,13 @@
                         <td id="password" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>-</td>
                         <td id="email" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>{{ user.email }}</td>
                         <td id="alias" class="editable" ng-click='manageUsers.editUser("row{{ i|e('js') }}")'>{{ user.alias|raw }}</td>
-                        <td id="token_auth_user" class="token_auth" data-token="{{ user.token_auth }}">{{ user.token_auth|slice(0, 8) }}…</td>
+                        <td id="token_auth_user" class="token_auth" data-token="{{ user.token_auth }}">
```

Comment:
@mattab Ideally we remove the tokens of all users here. 

It is off topic and does not belong to this PR and will be done as part of https://github.com/piwik/piwik/issues/6559 anyway at the latest but could be a good opportunity to remove them already in Piwik 3 from UsersManagement list. This is a security problem eg when there are 3 super users, and you can see the tokens of all other super users. Then you leave the company but still know the token of all other super users and users

@mattab maybe we can remove them already now in this PR as we want to improve security